admin/cipipeline/phase2.md

```diff
@@ -0,0 +1,45 @@
+# CI/CD Pipeline Phase 2
+
+## Pipeline graph
+
+![Drawio Pipeline Graph](phase2.drawio.png)
+
+## Currently Functional:
+
+### Prettier for Code Linting
+
+We are going to have members apply the `prettier` extension/package to every file that they edit for the project in order to maintain consistent code styling.
+
+Prettier is set up to auto-format any file that is being saved, ensuring that the code practices are uniform for all members
```

Comment:
Dhruv was asking about single vs double quotes and if it matters. I think he said he prefers double. If it matters let me know and I can update the config file, since not sure which one it defaults to now

Reply:
I dont think it really matters. I think the only time it makes a difference is whether we ever use single or double quotes within the string which i dont think we do

Reply:
Does it need to be consistent?